bootstrap/config/manager/manager.yaml

```diff
@@ -73,6 +73,13 @@ spec:
         image: controller:latest
         imagePullPolicy: Always
         name: manager
+        env:
+        - name: USE_INTERNAL_IMAGE_URL
+          value: "true"
+        - name: IMAGE_SERVICE_NAME
+          value: "assisted-image-service"
+        - name: IMAGE_SERVICE_NAMESPACE
+          value: "assisted-installer"
```

Comment:
Do we want to allow this to be configurable through the kustomize files somehow?

Reply:
I think we do. We should make it easy to customize and maybe default to false. @jianzzha do you have any suggestion about this?

Reply:
We can also set the default here too https://github.com/openshift-assisted/cluster-api-agent/pull/51#discussion_r1687128686

Reply:
I think this is ok.

To kustomize an env var like this, a hard coded index has to be used in the kustomization file. Say to kustomize  USE_INTERNAL_IMAGE_URL, in the kustomization file, index 0 will be used below
```- op: replace
        path: /spec/template/spec/containers/0/env/0/value
```

To make sure the kustomization file does not break when new env vars are added later,  new env vars have to be appended, not insert before the existing one - something to remember :)

Reply:
Great to know, thank you for your input!!

Reply:
I added an `overlays/ocp` folder to do a kustomize patch - what do you all think?